view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -187,6 +195,7 @@ define(
                     'scb': 0.0074,
                     'uob': 0.0064,
                     'mbb': 0,
+                    'ttb': 0,
```

Comment:
Make sure `TTB` has no interest. Typically zero percent interest is for Malaysia only.